crates/sui-indexer-alt-consistent-store/src/handlers/object_by_type.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use sui_indexer_alt_framework::{
+    pipeline::{sequential, Processor},
+    types::{base_types::VersionDigest, full_checkpoint_content::CheckpointData},
+};
+
+use crate::schema::{object_by_type::Key, Schema};
+use crate::store::{Connection, Store};
+
+use super::{checkpoint_input_objects, checkpoint_output_objects};
+
+pub(crate) struct ObjectByType;
+
+pub enum Value {
+    Put(Key, VersionDigest),
+    Del(Key),
+}
+
+#[async_trait]
+impl Processor for ObjectByType {
```

Comment:
This is pretty much the exact same code as `object_by_owner`, the only difference being the key, right?

Reply:
Yes, it is